kili/cli/project/create.py

```diff
@@ -63,7 +64,7 @@ def create_project(
     To build a Kili project interface, please visit: \n
     https://docs.kili-technology.com/docs/customizing-the-interface-through-json-settings
     """
-    kili = Kili(api_key=api_key, api_endpoint=endpoint)
+    kili = Kili(api_key=api_key, api_endpoint=endpoint, client_name=GraphQLClientName.CLI)
```

Comment:
for this one, I think the best option is to create a helper cli function ` get_kili_client` that instantiates the kili client with the correct client info:
```
def get_kili_client(api_key: str, api_endpoint: str):
      return Kili(..., client_name=GraphQLClientName.CLI)
```
this hides a detail, and is more future proof if we need to do more adaptations to the client